flink-core/src/main/java/org/apache/flink/core/security/FlinkSecurityManager.java

```diff
@@ -89,8 +89,8 @@ static FlinkSecurityManager fromConfiguration(Configuration configuration) {
 
         boolean haltOnSystemExit = configuration.get(ClusterOptions.HALT_ON_FATAL_ERROR);
 
-        // If no check is needed, return null so that caller can avoid setting security manager not
-        // to incur any runtime cost.
+        // If no check is needed, return null so that caller can avoid setting security manager and
```

Comment:
nit: security manager -> the security manager
imho:  I am not too worried about the text here - as the method javadoc seems to cover this. I would be inclined to remove this comment as it adds nothing to the javadoc. If we want to improve it, it would be better to have this comment explain why these if conditions indicate that we should not return a FlinkSecurityManager.


 - but if we change it here we should use the same words in the methods javadoc.

Reply:
Yes, the `javadoc` seems good enough. I think this inner comments just for readability.